style/utility.css

```diff
@@ -56,6 +52,12 @@ button {
   font-weight: bold;
 }
 
+.outline-btn:hover,
```

Comment:
You should also try to use components from your component library, like buttons..